src/Notifynder/Traits/Notifable.php

```diff
@@ -49,11 +49,11 @@ public function notifications()
     }
 
     /**
-     * Get the notifications Relationship.
+     * Get the notifications Relationship without any eager loading.
      *
      * @return \Illuminate\Database\Eloquent\Relations\HasMany|\Illuminate\Database\Eloquent\Relations\MorphMany
      */
-    public function getNotificationRelation()
+    private function getLazyLoadedNotificationRelation()
```

Comment:
Pls don't use `private` we are open for overrides and customization - so at least `protected`. Even if I don't get it 100% why we need it here? We can control the eager loading with the new parameter - so this is just a wrapper for `getNotificationRelation(false)`.